src/patternfly/components/MenuToggle/menu-toggle.scss

```diff
@@ -276,6 +277,14 @@
 
     height: 100%;
   }
+
+  .pf-c-text-input-group {
+    margin:
```

Comment:
This works, but it's probably better if we create local component vars for this. So this at the top of the scss

```
--pf-c-menu-toggle--c-text-input-group--MarginTop: calc(var(--pf-c-menu-toggle--PaddingTop) * -1);
--pf-c-menu-toggle--c-text-input-group--MarginRight: calc(var(--pf-c-menu-toggle--PaddingRight) * -1);
--pf-c-menu-toggle--c-text-input-group--MarginBottom: calc(var(--pf-c-menu-toggle--PaddingBottom) * -1);
--pf-c-menu-toggle--c-text-input-group--MarginLeft: calc(var(--pf-c-menu-toggle--PaddingLeft) * -1);
```

Then in this block:

```
margin: var(--pf-c-menu-toggle--c-text-input-group--MarginTop) var(--pf-c-menu-toggle--c-text-input-group--MarginRight) var(--pf-c-menu-toggle--c-text-input-group--MarginBottom) var(--pf-c-menu-toggle--c-text-input-group--MarginLeft);
```

This creates menu toggle component vars to restyle the text input group margin if a product wanted to do that, so that they don't have to write CSS.